src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -78,7 +79,7 @@ const useGroupInfo = (channelId = null) => {
     }
   };
   const memberName = () => {
-    if (anonUserEmojiName) return `Anonymous ${anonUserEmojiName}`;
+    if (anonUserEmojiName) return `${anonUserColorName} ${anonUserEmojiName}`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The modification in the `memberName` function is a good change as it now uses the `anonUserColorName` along with `anonUserEmojiName` to return the member name.
```diff
- if (anonUserEmojiName) return `Anonymous ${anonUserEmojiName}`;
+ if (anonUserEmojiName) return `${anonUserColorName} ${anonUserEmojiName}`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->